cluster-appliances/member-join

```diff
@@ -112,6 +115,16 @@ main() {
             _add_access_master_node "${cw_MEMBER_ip}" "${access_daemon_port:-25269}"
         fi
     fi
+    if [[ "${cw_INSTANCE_role}" == "master" && "${appliance_roles}" == *":storage:"* ]]; then
+        # We are the master and have just been notified of an ASM appliance instance
+        if handler_is_enabled cluster-www; then
```

Comment:
Due to changes in how we're handling `cluster-www` depending on what edition a user has access to, can you update this to the following please?

```
if [ "$(network_get_edition)" != "community" ] && handler_is_enabled cluster-www; then
```